ircdog.go

```diff
@@ -86,10 +86,10 @@ Options:
 	--tls-noverify      Don't verify the provided TLS certificates.
 	--listen=<address>  Listen on an address like ":7778", pass through traffic.
 	--hide=<messages>   Comma-separated list of commands/numerics to not print.
-	--no-italics        Don't use the ANSI italics code to represent italics.
-	--no-controls       Don't use the control character escapes.
+	-r --raw            Don't interpret IRC control codes.
```

Comment:
```suggestion
	-r --raw            Don't interpret IRC control codes when sending or receiving lines.
```

The text above this references the `--no-controls` option, which has been removed in favour of `--raw`. This line should also clarify that it applies to incoming **and** outgoing text.

The above text also says this, which needs to be updated to specify the `-e` escape (if the `-e` shorthand is allowed):
```
When the -r option is used, lines are displayed with the irc-go escapes
```